rgw/v2/tests/s3_swift/test_versioning_with_objects.py

```diff
@@ -305,6 +308,56 @@ def test_exec(config, ssh_con):
                                 "key_name: %s --> version_id: %s"
                                 % (version.object_key, version.version_id)
                             )
+                        if config.access_versioned_object:
+                            versions = bucket.object_versions.filter(
+                                Prefix=s3_object_name
+                            )
+                            i = 1
```

Comment:
@ckulal , what does the variable 'i' represent? can we have more readable variable name.

Reply:
done